docs/metrics.md

```
@@ -0,0 +1,354 @@
+# TYPE gunicorn_worker_state gauge
+gunicorn_worker_state{worker_id="worker_1_1234567890",state="running",timestamp="1640995200.123"} 1.0
+```
+
+### Master Metrics
+
+#### `gunicorn_master_worker_restarts_total`
+**Type**: Counter
+**Description**: Total number of worker restarts initiated by the master
+**Labels**: None
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)